### PR TITLE
Adding ability to show selected count in TableToolbar <?.?.x>

### PR DESCRIPTION
**Pull request by ryelo**
Adds ability to show selected count, results count, or both

Also updated the colors to closer match pagination 

screen shot 2019-03-08 at 1 22 56 pm

### CODE REVIEW

File: src/PresentationalComponents/TableToolbar/TableToolbar.js [hunk 1]
```diff
@@ -40,7 +45,8 @@ TableToolbar.propTypes = {
     isFooter: propTypes.bool,
     results: propTypes.number,
     children: propTypes.any,
-    className: propTypes.string
+    className: propTypes.string,
+    selected: propTypes.number
```
Comment (karelhala) [inline src/PresentationalComponents/TableToolbar/TableToolbar.js]: This should be number. Or `oneOfType([ propTypes.number, propTypes.string])`

Comment (ryelo) [reply src/PresentationalComponents/TableToolbar/TableToolbar.js]: @karelhala what should be number? `selected` is a number

Comment (karelhala) [reply src/PresentationalComponents/TableToolbar/TableToolbar.js]: Yeah, my bad. I saw string.

**Pull request merged**